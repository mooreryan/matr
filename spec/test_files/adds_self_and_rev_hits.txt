source	target	score
ryan	amelia	99
