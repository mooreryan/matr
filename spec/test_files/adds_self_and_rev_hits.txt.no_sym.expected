	amelia	ryan
amelia	100	0
ryan	99	100
